src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -51,14 +61,15 @@
  *
  * @author Dmitry Zaytsev (dmitry.zaytsev@gmail.com)
  * @version $Id$
- * @since 0.15.2
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  * @checkstyle MultipleStringLiteralsCheck (500 lines)
  * @todo #306:30min At the moment we don't support HTTP
- *  persistent connections. Would be great to implement
- *  this feature. BkBasic.accept should handle more
- *  than one HTTP request in one connection.
+ * persistent connections. Would be great to implement
```

Comment:
@alexey-krylov Travis build failed because of invalid format in the puzzle. You have to add a space from second line

>/home/travis/.rvm/gems/ruby-1.9.3-p551/gems/pdd-0.15.4/lib/pdd/source.rb:144:in `rescue in puzzles': ["in /home/travis/build/yegor256/takes/src/test/java/org/takes/http/BkBasicTest.java", #<PDD::Error: Space expected>] (PDD::Error)